services-camel/src/main/java/edu/unc/lib/dl/services/camel/fulltext/FulltextProcessor.java

```diff
@@ -97,12 +97,13 @@ public void process(Exchange exchange) throws Exception {
 
         String fedoraUri = (String) in.getHeader(FCREPO_URI);
         String binaryPath = (String) in.getHeader(CdrBinaryPath);
+        String mimetype = (String) in.getHeader(CdrFcrepoHeaders.CdrBinaryMimeType);
```

Comment:
What about having `FulltextProcessor.allowedTextType` filter out json files instead of doing so at extraction time?

Reply:
Hmm, looks like they should already be filtered out, as long as they get the right mimetype from fits. If they get the text/plain mimetype they'll pass the test in this pr and still have their text extracted. Seems this PR can just be closed.